src/main/docker/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM eclipse-temurin:17-jre-alpine
```

Comment:
move the docker folder to root directory

Reply:
@caominhnhut Moved to root directory on latest commit